build.sh

```diff
@@ -57,7 +57,13 @@ check_prereqs()
     hash cmake 2>/dev/null || { echo >&2 "Please install cmake before running this script"; exit 1; }
     
     # Check for clang
-    hash clang 2>/dev/null || { echo >&2 "Please install clang before running this script"; exit 1; }
+    OS=`uname`
+    if [ $OS = "Linux" ]; then
```

Comment:
The clang naming is not a general Linux problem. It is interesting that e.g. in my Ubuntu, all the llvm-xxx commands have the 3.5 suffix, but the clang doesn't.